lib/src/components/NxTag/NxTag.scss

```diff
@@ -7,17 +7,18 @@
 @use '../../scss-shared/nx-text-helpers';
 @use '../../scss-shared/nx-container-helpers';
 @use '../../scss-shared/nx-hide-checkbox-helper';
+@use '../../scss-shared/nx-tag-variables';
 
 .nx-tag {
   @include nx-text-helpers.semi-bold;
   @include nx-container-helpers.container-horizontal;
 
   align-items: center;
   background-color: var(--nx-selectable-color-dark);
-  border: 1px solid var(--nx-selectable-color-dark);
+  border: var(--nx-border-width) solid var(--nx-selectable-color-dark);
```

Comment:
I believe the padding below needs to also use this variable rather than `1px`

Reply:
> I believe the padding below needs to also use this variable rather than `1px`

Changed here: 53af6b3256b4a0d0845640be2d5b8b94cbbb6832